pkg/webhook/mutation/pod/metadata/workload_test.go

```diff
@@ -211,3 +236,13 @@ func createTestWorkloadInfo() *workloadInfo {
 		name: "test",
 	}
 }
+
+func createFailK8sClient() client.Client {
```

Comment:
i think it's good practice to mark helpers as `t.Helper` https://pkg.go.dev/testing#T.Helper
>Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped. Helper may be called simultaneously from multiple goroutines.

Reply:
example https://github.com/dynatrace/dynatrace-operator/blob/cba0da7700f4adcec9fd44fb52bd0b81ed2005f2/test/helpers/registry/registry.go#L28-L36

Reply:
surprised the linter is not complaining about it, but IMO it's fine here, it just creates an object 🤷

Reply:
it's up to you 👍 not blocking comment.

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3783/commits/218c5a125d1cbb6b714042d24751ba5aa5232f0b

I think its a good idea 👍